### PR TITLE
Add try catch to ph5avialbility.

### PR DESCRIPTION
**Pull request by timronan**
Thank your for contributing to PH5

Before submitting a PR, please review the pull request guidelines:
https://github.com/PIC-IRIS/PH5/blob/master/CONTRIBUTING.md#submitting-a-pull-request

### Checklist
- [ ] This PR is not directly related to an existing issue (which has no PR yet).
- [ ] All tests pass.
- [ ] Any new or changed features have are documented.
- [ ] Changes have been added to `CHANGELOG.txt` .
- [ ] First time contributors have added your name to `CONTRIBUTORS.txt` .

### CODE REVIEW

File: ph5/clients/ph5availability.py [hunk 1]
```diff
@@ -756,18 +756,30 @@ def get_availability(self, station='*', location='*',
                             das_info[ph5_das] = {}
                         k = (channum,  ph5_start_epoch,
                              ph5_stop_epoch, ph5_sample_rate)
-                        if k in das_info[ph5_das].keys():
-                            Das_t = das_info[ph5_das][key]
```
Comment (damhuonglan) [inline ph5/clients/ph5availability.py:760]: I had a mistake here. Please use `Das_t = das_info[ph5_das][k]`

File: ph5/clients/ph5availability.py [hunk 2]
```diff
@@ -756,18 +756,30 @@ def get_availability(self, station='*', location='*',
...
-                        else:
-                            Das_t = self.ph5.query_das_t(
-                                ph5_das,
-                                chan=channum,
-                                start_epoch=ph5_start_epoch,
-                                stop_epoch=ph5_stop_epoch,
-                                sample_rate=ph5_sample_rate,
-                                sample_rate_multiplier=ph5_multiplier,
-                                check_samplerate=False)
-                            das_info[ph5_das][key] = Das_t
```
Comment (damhuonglan) [inline ph5/clients/ph5availability.py:770]: Also here `das_info[ph5_das][key] = Das_t`.
Then you wont get KeyError anymore.

**Review by damhuonglan — CHANGES_REQUESTED** [review]

**Comment by damhuonglan** [comment]
@timronan I see that you still didn't finish this branch. I will finish this and ask you to have the final review.

**Comment by damhuonglan** [comment]
@timronan I fixed the code but can't request you to review it because you are the owner of this PR. Please has a look and leave a comment here. Thank you.

**Comment by damhuonglan** [comment]
@timronan I fixed the key but can't request you to review because you're the owner of the  PR. Please have a look and leave your review here. Thanks.

**Comment by timronan** [comment]
Thanks for fixing that. I am merging now.

File: ph5/clients/ph5availability.py (hunk 1 above)
Comment (timronan) [reply ph5/clients/ph5availability.py:760]: Resolved

**Comment by timronan** [comment]
@damhuonglan Please re-review and resolve the conversation so I can merge it.

**Review by damhuonglan — APPROVED** [review]
It's good to be merged now.

**Pull request merged**